setup.cfg

```diff
@@ -34,6 +34,9 @@ install_requires =
   setuptools_scm
   zprocess>=2.18.0
 
+[options.extras_require]
+docs = PyQt5; Sphinx==3.0.1; sphinx-rtd-theme==0.4.3; recommonmark==0.6.0; m2r==0.2.1
```

Comment:
Does this syntax work? The semicolons usually split a requirement from an environment marker like `importlib_metadata; python_version < '3.8'` so I would have expected this to require spaces or newlines between the requirements.

Reply:
https://stackoverflow.com/a/49820357 and official docs within.
But I might try a multiline version as it'll track changes better in git

Reply:
Ah, fair enough. I guess things evolved from setup.py in a way that made the different config options a bit inconsistent.

Reply:
resolved in f2d68573a5c7265d7661844d933d4a442b406ed6